mesh_map/src/mesh_map.cpp

```diff
@@ -436,7 +494,9 @@ void MeshMap::combineVertexCosts(const rclcpp::Time& map_stamp)
     float min, max;
     mesh_map::getMinMax(costs, min, max);
     const float norm = max - min;
-    const float factor = 1.0; // TODO how to declare param for each plugin? Needs to be done after plugins are loaded, which happens when the map gets loaded. Who calls readMap() and when?
+    const float factor = 1.0;
+    // TODO how to declare param for each plugin? 
+    // Needs to be done after plugins are loaded, which happens when the map gets loaded. Who calls readMap() and when?
```

Comment:
Is this outdated?

Each plugin needs to define its own parameters. They do that in the `initialize()` method, but I see that you already found that method.

Reply:
outdated. removed